src/SkipOnEmptyCallback/SkipNever.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator\SkipOnEmptyCallback;
+
+final class SkipNever
```

Comment:
Maybe the following would be better name. What do you think?

```suggestion
final class SkipNone
```

Reply:
Yes it's better because it's consistent with others - it highlights what is skipped and not when.